src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star stars_star1"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
```

Comment:
Don't need to add additional class here. You have a modificator for stars, so you need to use this for making stars active in css.

```suggestion
    <div class="stars stars--3">
      <div class="stars_star"></div>
      <div class="stars_star"></div>
      <div class="stars_star"></div>
      <div class="stars_star"></div>
      <div class="stars_star"></div>
```